src/test/isolation2/expected/vacuum_progress_column.out

```diff
@@ -341,12 +341,31 @@ select relid::regclass as relname, phase, heap_blks_total, heap_blks_scanned, he
  Success:        
  Success:        
 (3 rows)
+
+-- we shouldn't proceed until FTS probe has bumped the version
```

Comment:
I am afraid this comment seems to be a bit misleading. Perhaps you mean status_version in  FtsVersion in FtsProbeInfo, may be it's better to specify it.
Also, the comment says about bumping the version, but nothing with the code below related to it, instead it waits for the mirror to be down. I suggest to rephrase this comment to match the code.